cms/djangoapps/contentstore/views/course.py

```diff
@@ -56,21 +48,6 @@
     reverse_library_url,
     reverse_usage_url,
     reverse_url,
-<<<<<<< HEAD
-    remove_all_instructors,
-)
-from models.settings.course_details import CourseDetails, CourseSettingsEncoder
-from models.settings.course_grading import CourseGradingModel
-from models.settings.course_metadata import CourseMetadata
-from util.json_request import expect_json
-from util.keyword_substitution import get_keywords_supported, substitute_keywords_with_data
-from util.string_utils import _has_non_ascii_characters
-from student.auth import has_studio_write_access, has_studio_read_access
-from .component import (
-    SPLIT_TEST_COMPONENT_TYPE,
-    ADVANCED_COMPONENT_TYPES,
-=======
->>>>>>> release-2016-02-09
```

Comment:
It looks like some of this was refactored.
All of these, except `render_to_string ` and `util.keyword_substitution` (see below) were duplicated elsewhere.